src/index.js

```diff
@@ -105,21 +105,65 @@ const createWindows = () => {
   captureWindow.on("resize", (event) =>
     updateMain(null, captureWindow.getSize())
   );
-  captureWindow.on("move", (event) =>
-    updateMain(captureWindow.getPosition(), null)
-  );
+  captureWindow.on("move", (event) => {
+    updateMain(captureWindow.getPosition(), null);
+    checkWindowBounds(captureWindow);
+    determineScreenToCapture();
+  });
```

Comment:
I think calling `determineScreenToCapture()` frequently caused flickering on some systems which is why I moved this to the `moved` event. But it seems this event is [not available on linux](https://www.electronjs.org/docs/latest/api/browser-window#event-moved-macos-windows). Could you add a timer that will trigger the function like this? I think we can then get rid of the `moved` event altogether!
```ts
  let moveTimer = undefined;
  captureWindow.on("move", (event) => {
    clearTimeout(moveTimer);
    moveTimer = setTimeout(() => {
      checkWindowBounds(captureWindow);
      determineScreenToCapture();
    }, 100);
    updateMain(captureWindow.getPosition(), null);
  });
```

Reply:
I like it - something like a debounce